Systematics/plugins/PhotonSigEoverEShift.cc

```diff
@@ -57,7 +57,7 @@ namespace flashgg {
                     std::cout << beforeSigEoE << " ";
                     std::cout << std::endl;
                 }
-                y.shiftSigmaEOverEValueBy( shift ); 
+                y.shiftSigmaEOverEValueBy( shift, 0.00 ); 
```

Comment:
Is this a bugfix or a compatibility issue?

Reply:
Compatibility, the function in Photon.cc is changed. https://github.com/threiten/flashgg/commit/4d6f19d5e63b06850fb2c0a1339078562f658e54

Reply:
ok thanks